src/constants/index.ts

```diff
@@ -2,6 +2,8 @@ export const OPERATOR_COOKIE = 'fdbt-operator';
 
 export const FARETYPE_COOKIE = 'fdbt-fareType';
 
+export const PASSENGERTYPE_COOKIE = 'fdbt-passengerType';
```

Comment:
Just a thought would this read better as PASSENGER_TYPE_COOKIE. I know the other types have it but seems quite long

Reply:
Changed cookie to PASSENGER_TYPE_COOKIE.